docs/tutorial/1/README.md

```diff
@@ -34,8 +34,8 @@ You should see the following logs:
 ```
 Local: index: Indexing local folder...
 Local: index: indexed file: 1.txt
-Local: index: indexed file: 3.jpg
 Local: index: indexed file: 2.md
+Local: index: indexed file: 3.jpg
```

Comment:
I'd prefer to leave this order in the docs as is, as it is actually representing the fact that mtriage can run analysis/selection of different elements in parallel (and thus the order of logs here is not strictly sequential).